### DISCUSSION TITLE
Next.js replaces the ip defined in env.local with it's hostname, is there a way to change this?

### DISCUSSION THREAD

**Discussion by piperun** [vercel/next.js#25830] [Help]
Essentially when I use .env.local to store say my database credentials and connection info, next.js will return the hostname of said IP when used instead of the IP, but since I'm only hosting this locally for now I have not bothered setting up any DNS server.

Is there any way to change this behavior?

**Comment by leerob** [comment]
I'm not sure I'm following – an environment variable from `.env.local` is getting overwrote somehow? Could you provide a reproducible example?

**Reply by piperun** [reply]
Sorry about that.
Essentially if my database has the hostname: "wooden_tiger" and it's ip is 192.168.1.13
And this info is stored in .env.local:
```
IP=127.0.0.1
DB_HOST=db://$IP:1234
```
If we then print out said enviromental var(s) using `console.log()`, it'll show `DB_HOST=db://wooden_tiger:1234` instead of `DB_HOST=db://192.168.1.13:1234`.

**Reply by leerob** [reply]
Why not just use one environment variable?

**Reply by piperun** [reply]
That certainly works, I was hoping to use IP var as a more flexible way of changing IP without having to worry about accidentally removing more than I wanted.
